### PR TITLE
Remove library dependency on micro-lens

### PR DESCRIPTION
**Pull request by duog**
Note that the name of the module no longer seems appropriate. I would
recommend it be changed to Data.Reocrd.Lens.

ref #26

### CODE REVIEW

**Comment by edsko** [comment]
Maybe we should rename it `.VanLaarhoven` then? My original motivation for the prefix was to distinguish it from more opaque representations like the WT optics.

File: src/Data/Record/Generic/Lens/Micro.hs [hunk 1]
```diff
@@ -9,6 +9,14 @@
 {-# LANGUAGE TypeOperators         #-}
 {-# LANGUAGE QuantifiedConstraints #-}
 
+-- | van Larhoovan lenses for large records.
```
Comment (edsko) [inline src/Data/Record/Generic/Lens/Micro.hs:12]: Laarhoven :)

Comment (duog) [reply src/Data/Record/Generic/Lens/Micro.hs]: Awk. force pushed a fix.

File: src/Data/Record/Generic/Lens/Micro.hs [hunk 2]
```diff
@@ -9,6 +9,14 @@
 {-# LANGUAGE TypeOperators         #-}
 {-# LANGUAGE QuantifiedConstraints #-}
 
+-- | van Laarhovan lenses for large records.
```
Comment (edsko) [inline src/Data/Record/Generic/Lens/Micro.hs:12]: Getting closer :p

**Comment by duog** [comment]
> Maybe we should rename it `.VanLaarhoven` then? My original motivation for the prefix was to distinguish it from more opaque representations like the WT optics.

Sure. Although hard to spell, perhaps `VL`?

**Comment by edsko** [comment]
> Sure. Although hard to spell, perhaps VL?

Haha, well, you made that point just now I suppose :) I guess it's easier for a Dutch person like me :) Yeah, `.VL` works for me.

**Comment by duog** [comment]
I'll rename the module, should I bump the major version as well?

**Comment by edsko** [comment]
Nah, don't worry, I'll do that on release. There are some other breaking changes that I am planning to make as well. 

Thanks! :)

**Comment by edsko** [comment]
Should I merge? Did you run the test suite? Not sure if there are imports to be fixed there.

I should really set up CI for this repo :)

**Comment by duog** [comment]
I've run the test suite. I'm happy for you to merge now

**Comment by edsko** [comment]
Thanks! Appreciate it :)

**Pull request merged**